operations/node_management/Add_Remove_Replace_NCNs.md

```diff
@@ -1,24 +1,65 @@
-# Alpha Framework to Add, Remove or Replace NCNs
+# Alpha Framework to Add, Remove, Replace or Move NCNs
+
+Add, Remove, Replace or Move worker, storage or master node (NCN). Use this procedure in the event that:
+
+- Worker, storage or master nodes are being replaced and the MAC address is changing.
+- Worker or storage nodes are being added.
+- Worker, storage or master nodes are being moved to a different cabinet.
+
+The following workflows are available:
 
 - [Prerequisites](#prerequisites)
 - [Add Worker, Storage or Master NCNs](#add-worker-storage-master)
+   - [Add NCN Prerequisites](#add-ncn-prerequisites)
+   - [Add NCN Procedure](#add-ncn-procedure)
 - [Remove Worker, Storage or Master NCNs](#remove-worker-storage-master)
+   - [Remove NCN Prerequisites](#remove-ncn-prerequisites)
+   - [Remove NCN Procedure](#remove-ncn-procedure)
 - [Replace or Move Worker, Storage or Master NCNs](#replace-worker-storage-master)
 
 <a name="prerequisites"></a>
 ### Prerequisites
 
 The system is fully installed and has transitioned off of the LiveCD.
 
-1. preq.py placeholder
+All activities required for site maintenance are complete.
+
+The latest docs-csm RPM has been installed on the master nodes.
+
+1. Run ncn_add_remove_replace_ncn_pre-req.py to adjust the network.
+
+   ```bash
+   ncn-m# cd /usr/share/doc/csm/scripts/operations/node_management/Add_Remove_Replace_NCNs/
+   ncn-m# ./ncn_add_remove_replace_ncn_pre-req.py 
+   ```
+
+   1. Script will ask 3 questions:
+
+      1. How many NCNs would you like to remove?  Do not include NCNs to be add or moved.
```

Comment:
I'm not sure how this looks once you run the script, but it could be helpful to provide some more context on answering these questions. 

Here are some things I was curious about:

Do you input the specific xnames? Or do you just specify the number of NCNs?
Is there anything specific that needs to be done if you are not removing any NCNs?